react/FilePicker/FilePickerBodyItem.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import filesize from 'filesize'
+import { makeStyles } from '@material-ui/core/styles'
+
+import { models } from 'cozy-client'
+
+import ListItem from '../MuiCozyTheme/ListItem'
+import ListItemIcon from '../MuiCozyTheme/ListItemIcon'
+import ListItemText from '../ListItemText'
+import Divider from '../MuiCozyTheme/Divider'
+import Icon from '../Icon'
+import FileTypeText from '../Icons/FileTypeText'
+import FileTypeFolder from '../Icons/FileTypeFolder'
+import Checkbox from '../Checkbox'
+import Radio from '../Radio'
+import { useI18n } from '../I18n'
+
+const {
+  file: { isDirectory, isFile }
+} = models
+
+const useStyles = makeStyles(() => ({
+  verticalDivider: {
+    height: '2rem',
+    display: 'flex',
+    alignSelf: 'auto',
+    alignItems: 'center',
+    marginLeft: '0.5rem'
+  },
+  listItemIcon: {
+    marginLeft: '1rem'
+  }
+}))
+
+const FilePickerBodyItem = ({
+  file,
+  fileTypesAccepted,
+  multiple,
+  handleChoiceClick,
+  handleListItemClick,
+  filesIdsSelected,
+  hasDivider
+}) => {
+  const classes = useStyles()
+  const { f } = useI18n()
+  const hasChoice =
```

Comment:
useMemo ?

Reply:
I don't think that operation deserves a useMemo. 

isFile and isDirectory should not cost big time of computation.

https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
Je ne pense pas que ça soit pertinent dans ce genre de cas, je suis presque sûr que la mémoïsation sera plus couteuse que ce test.